src/clj/clojurians_log/db/queries.clj

```diff
@@ -55,3 +55,13 @@
          [?chan :channel/name ?chan-name]]
        db
        name))
+
+(defn user-names
+  [db names]
+  (d/q '[:find ?ids ?username
+         :in $ [?ids ...]
```

Comment:
Since you're destructuring this collection the variable will refer to a single `?id`, not multiple `?ids`